src/components/toplayer/_toplayer.scss

```diff
@@ -84,10 +85,11 @@ $includeHtml: false !default;
     }
 
     &__close {
-      @include fixText(24px, -4px);
       color: $toplayerCrossColor;
+      font-size: 32px;
```

Comment:
Do we need it?
I am still confused by the framework of moving things to variables.